mobify/platform-scaffold
# [WEB-1396] Rename `[id].js` named chunks to their template name

Code split chunks are currently named `[id].js` and this isn't ideal. This is a PR to change that

 **JIRA**: https://mobify.atlassian.net/browse/WEB-1396
 **Linked PRs**: N/A

## Changes
- Update to `webpack 2.6.1`
- Name chunks to their template names

## Todos

## How to test-drive this PR
- Remove existing `node_modules` folder:
```
$ rm -rf /web/node_modules
```
- Run `npm install`
- Run `npm run dev`
- Observe the named chunks.

## Results
| Before | After |
| --- | --- |
|![screen shot 2017-06-14 at 4 33 23 pm](https://user-images.githubusercontent.com/10932540/27159046-47027a1a-511f-11e7-8815-1a7e82714b03.png) | ![screen shot 2017-06-14 at 4 28 43 pm](https://user-images.githubusercontent.com/10932540/27158942-98ea9e76-511e-11e7-8483-eb7d58c40704.png)|
